packages/mobile/src/identity/privateHashing.ts

```diff
@@ -84,6 +89,11 @@ function* doFetchPhoneHashPrivate(e164Number: string) {
   return details
 }
 
+export function* balanceSufficientForQuotaRetrieval() {
+  const userBalance = yield select(stableTokenBalanceSelector)
+  return new BigNumber(userBalance).isGreaterThanOrEqualTo(ODIS_MINIMUM_DOLLAR_BALANCE)
```

Comment:
Can you confirm the user balance is stored in dollars and not wei? It may be, I just can't remember.

Reply:
all other stableTokenBalanceSelector calls treat is as if it is stored in dollars